lib/matplotlib/patches.py

```diff
@@ -3505,11 +3513,13 @@ class CurveFilledB(_Curve):
 
         def __init__(self, head_length=.4, head_width=.2):
             """
-            *head_length*
-              length of the arrow head
+            Parameters
+            ----------
+            head_length : float, optional, default : .4
```

Comment:
cc @NelleV 

I don't we should have two colons in the same sentence. It looks weird to me (but it's not a deal breaker). I think this should read:

```
headhead_length : float, optional (default is 0.4)
```

The other question is if these parameters are actually optional. What I mean by that is if you said `head_length=None`, would this still work by falling back to the default value? If not, then the parameter isn't actually, optional, just named.